index.html

```diff
@@ -1198,23 +1202,40 @@
         NDEFRecordData data;
       };
     </pre>
+    <p>
+      <a>NDEFRecord</a> objects have an associated <dfn id="ndefrecord-bytes">bytes</dfn>
+      (a <a>byte sequence</a>) set on creation that represents the payload data of the
+      <a>NDEF record</a>, which is `null` if there was no data in the
```

Comment:
data is plural, so I think it should be "were"

Reply:
Done.